docs/design/access.md

```diff
@@ -130,7 +130,7 @@ simple and enterprise profiles.  System should be flexible enough for
 knowledgeable users to create intermediate profiles, but K8s developers should
 only reason about those two Profiles, not a matrix.
 
-Features in this doc are divided into "Initial Feature", and "Improvements".
+Features in this doc are divided into "Initial Features", and "Improvements".
```

Comment:
This change is OK, but the doc has moved to the community repo, so we can't take this PR anyway.